timeout.c

```diff
@@ -35,7 +35,7 @@
 
 #include <errno.h>     /* errno */
 
-#include <sys/queue.h> /* TAILQ(3) */
+#include <queue.h> /* TAILQ(3) */
```

Comment:
This doesn't exist on my archlinux system.

Reply:
This is a change that should not be in this pull request my bad.
I'm using a different version ( freebsd) of queue.h instead of sys/queue.h
I'm using this branch in some project.
For now until i fix this you should just use sys/queue.h